.github/workflows/very_good_cli.yaml

```diff
@@ -31,7 +31,7 @@ jobs:
         run: flutter pub get
 
       - name: Format
-        run: dart format --set-exit-if-changed .
+        run: dart format --set-exit-if-changed lib test
 
       - name: Analyze
```

Comment:
This file has a dart test with the args: `-x e2e`. We should remove that since that tag `e2e` doesnt exist on the CLI anymore.